apps/pie-docs/README.md

```diff
@@ -61,7 +61,7 @@ Our primary concerns are:
 ### Page testing
 We have route navigation tests that ensure all existing pages can be correctly navigated to. When you add new pages, these tests will fail as there are new unexpected pages (this is by design).
 
-In order to fix the tests, you will need to register the routes to your newly added pages by running: `yarn test:generate-routes`. This will update the JSON file named `expected-routes.snapshot.json` with the URLs to your new page.
+In order to fix the tests, you will need to register the routes to your newly added pages by running from the root of the monorepo: `yarn test:generate-routes`. This will update the JSON file named `expected-routes.snapshot.json` with the URLs to your new page.
```

Comment:
do you think this is easier to read?

```suggestion
In order to fix the tests, you will need to register the routes to your newly added pages by running: `yarn test:generate-routes` from the root of the monorepo. This will update the JSON file named `expected-routes.snapshot.json` with the URLs to your new page.
```

Reply:
sounds good!